src/kbmod_wf/task_impls/reproject_wu.py

```diff
@@ -74,13 +74,13 @@ def __init__(
         # Default to 8 workers if not in the config. Value must be 0<num workers<65.
         self.n_workers = max(1, min(self.runtime_config.get("n_workers", 8), 64))
 
+        #! In the long run, we likely won't have the URI files to start from
+        #! So we'll need to rethink how we get these parameters.
```

Comment:
These new comments are the biggest open question at this point - Dino is recommending a dynamic approach to gathering this information. Looking into that now.